src/index.tsx

```diff
@@ -10,7 +10,7 @@ type ExcaliburTextElement = ExcaliburElement & {
   type: "text";
   font: string;
   text: string;
-  measure: TextMetrics;
```

Comment:
TextMetrics doesn't serialize using JSON.stringify because the elements are getters. In any case, we only need actualBoundingBoxAscent in order to do the drawing.